test/cpp_extensions/libtorch_agnostic_extension/libtorch_agnostic/csrc/cuda_kernel.cu

```diff
@@ -0,0 +1,34 @@
+#include "kernel.h"
+
+#include <torch/csrc/stable/library.h>
+#include <torch/csrc/stable/tensor.h>
+#include <torch/csrc/stable/ops.h>
+#include <c10/cuda/CUDAException.h>
```

Comment:
This one is not okay to include. C10_CUDA_KERNEL_LAUNCH_CHECK() cannot be used.

Reply:
This migration will be needed also for https://github.com/pytorch/audio/pull/4115.

Meanwhile, skip the check.

Reply:
Ah, sorry for that, the include was meant to be removed!